DesignSafe-CI/portal
# Task/des 2631 apps breadcrumb

## Overview: ##
Add breadcrumb for apps

## PR Status: ##


## Related Jira tickets: ##

* [DES-2631](https://tacc-main.atlassian.net/browse/DES-2631)

## Summary of Changes: ##
Added AppsBreadcrumb to workspace
Then added it to WorkspaceBaseLayout 
Within WorkspaceBaseLayout, there's some data manipulation to make the breadcrumb display as expected. 

## Testing Steps: ##
1. Starting at https://designsafe.dev/rw/workspace, Home / Use DesignSafe / Tools & Applications should appear
2. Use DesignSafe is not a link, should not be clickable
3. Clicking Job Status changes the url to jobs/history, but the breadcrumb to / Job Status
4. Clicking an app changes the url to /app-id, but the breadcrumb should show the bundle title (Ex: / OpenSees)
5. Home should go to designsafe.dev (not working right now)

## UI Photos:
<img width="507" alt="Screen Shot 2024-04-24 at 2 02 21 PM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/830bdb5a-3988-4ce5-bf4c-4699afe63a68">




## Notes: ##
Home doesn't go home. I'm not sure exactly how to do that.
